Formula/liquidprompt.rb

```diff
@@ -3,10 +3,19 @@ class Liquidprompt < Formula
   homepage "https://github.com/nojhan/liquidprompt"
   url "https://github.com/nojhan/liquidprompt/archive/v_1.10.tar.gz"
   sha256 "5a1e01b86e81002ac20c61cbd96f5fd276db7f3fa09ef220f3db0fb7aa614e24"
-  head "https://github.com/nojhan/liquidprompt.git", :branch => "develop"
 
   bottle :unneeded
 
+  head do
```

Comment:
This doesn't need moving, or the explicit `version`. You just need to remove the `:branch => "develop"` from the original line here.